docs/flexibility/trigger-updates.md

```diff
@@ -56,6 +58,7 @@ The following status variables are available:
 
 | Variable name                         | Description |
 |---------------------------------------|-------------|
+| `Open_tables_with_triggers` | The current number of TABLE instances with fully-loaded triggers in the Table cache. |
```

Comment:
Perhaps it is better to be consistent with description of other status variables in this table and say "... in the `table_open_cache`." instead ?